pixeltable/plan.py

```diff
@@ -374,6 +390,21 @@ def create_update_plan(
             recomputed_cols.update(idx_val_cols)
             # we only need to recompute stored columns (unstored ones are substituted away)
             recomputed_cols = {c for c in recomputed_cols if c.is_stored}
+
+        for col in recomputed_cols:
+            if col.value_expr is not None and not col.value_expr.is_valid:
```

Comment:
Move this into a helper class method and call from both places?

Reply:
Done.